src/app/components/msel-info/msel-info.component.ts

```diff
@@ -355,6 +355,19 @@ export class MselInfoComponent implements OnDestroy {
     }
   }
 
+  hasStartTime(): boolean {
+    return this.msel.startTime && this.msel.startTime.valueOf() > 0;
+  }
+
+  toggleStartTime(event: any) {
+    this.msel.startTime = event.checked
+      ? new Date()
+      : new Date('0001-01-01T00:00:00');

```

Comment:
Why the arbitrary date?

Reply:
That is the "nullish" date value when there is no date.